specifications/api/swagger.yaml

```diff
@@ -4068,3 +4133,25 @@ paths:
 
         Return the current settings.
       responses: *responsesGetSettings
+
+  /smash/health:
+    post:
+      operationId: getUriSmashHealth
+      tags: ["Utils"]
+      summary: SMASH health
+      description: |
+        Get health status of given SMASH uri.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema: *ApiSmashServer
+      responses: *responsesPostSmashHealth
```

Comment:
A `POST` request is awkward here, there's no resources being created. May we stick to a more restful approach and offer instead one single endpoint `GET /v2/smash/health` which also accepts an optional query parameter `url` or type `ApiSmashServer`; when provided, we'll fetch health from this URL, when omitted, we'll fetch health from the currently configured smash server if any.

Reply:
I was just trying to avoid url encoding as query parameter, since it's pesky.

Reply:
I'd favor pesky url-encoded query parameter over awkward non restful post request :blush: